tables/rubin_early_pp_scenario.tex

```diff
@@ -15,16 +15,19 @@
         {\parbox{0.5\linewidth}{\vspace{0.6cm} \textbf{Data Product}}}  &   
-        { \makecell{ \textbf{Commissioning } \\ \textbf{(System Optimization \& SV Surveys)} }}  & 
+        { \makecell{ \textbf{LSSTCam Commissioning }  }}  & 
         {\makecell{\textbf{Year 1 Survey} \\ \textbf{Operations} }} 
          \\[10pt] \cline{2-3} \hline
 
-\textbf{Alerts}     & --- &   --- \\  \arrayrulecolor{gray}\hline
-\textbf{PP Processed Visit Images}     & Commissioning of the PP image differencing and incremental template building will begin shortly after System First Light. Prompt image release is embargoed during commissioning.  &   Access to unvetted processed visit images as prompt products in the first 6 months of the LSST is still TBD. PP PVIs available after DP2.     \\  \arrayrulecolor{gray}\hline
-\textbf{PP Difference Images}     & Difference imaging will be somewhat limited, since the image template sky coverage will be sparse. Prompt image release is embargoed during commissioning.  &     Difference imaging will steadily increase as incremental template building increases the templates available. Prompt access to unvetted PP difference images is still TBD. PP diff images available after DP2.    \\\hline
-\textbf{PP Catalogs}    &   PPDB likely unavailable for query. &  PPDB available for query. \\ 
- (DIASources, DIAObjects, DIAForced Sources)  & & \\\hline
-\textbf{PP SSP Catalogs}   &   Alert volume and latency will improve throughout the commissioning period. Aiming for ``near-live'' brokered Alert stream by the end of LSSTCam SV.  &   Difference imaging will steadily increase as incremental template building increases the templates available. Prompt access to unvetted PP difference images is still TBD. PP diff images available after DP2. \\  \hline
+\textbf{Alerts}     &  Alert volume and latency will improve throughout the commissioning period. Aiming for ``near-live'' brokered Alert stream by the end of LSSTCam Commissioning.  &  
+Continued ramp up of the alert stream contingent on the availability of templates.  Alerts expected to reach near full volume and fidelity after DR1.  Alert stream latency in year 1 is 120 seconds. \\  \arrayrulecolor{gray}\hline
+%
+\textbf{PP Processed Visit Images}     & Commissioning of the PP image differencing and incremental template building. Prompt image release is embargoed during commissioning (\S~\ref{ssec:impact}).  &   Access to unvetted processed visit images as prompt products in the first 6 months of the LSST is TBD.      \\  \arrayrulecolor{gray}\hline
+\textbf{PP Difference Images}     & Difference imaging will be somewhat limited, since the image template sky coverage will be sparse. Prompt image release is embargoed during commissioning (\S~\ref{ssec:impact}). &     Difference imaging will steadily increase as incremental template building increases the templates available. Prompt access to unvetted PP difference images is TBD.    \\\hline
```

Comment:
```suggestion
\textbf{PP Difference Images}     & Difference imaging will be somewhat limited, since the image template sky coverage will be sparse. Prompt image release is embargoed during commissioning (\S~\ref{ssec:impact}). &     Difference imaging will steadily increase as incremental template building increases the templates available. Access to PP difference images in the first 6 months of LSST is TBD.    \\\hline
```